http-add-on/Chart.yaml

```diff
@@ -11,7 +11,7 @@ kubeVersion: ">=v1.23.0-0"
 # to the chart and its templates, including the app version. This is incremented at chart release time and does not need
 # to be included in any PRs to main.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.10.0
+version: 0.10.1
```

Comment:
please don't bump chart version, there is a release process for addon
https://github.com/kedacore/http-add-on/blob/main/RELEASE-PROCESS.md

examples:
- https://github.com/kedacore/charts/pull/721
- https://github.com/kedacore/charts/pull/737

Reply:
@mthemis-provenir also please remove the archive from `/docs`